docs/specs/xref.yml

```diff
@@ -20,3 +20,23 @@ outputs:
   build.log: |
     ["info","at-uid-not-found","Cannot find uid 'a' using xref '@a'","docs/a.md"]
   build.manifest:
+---
+# Restore xref map from configured url and use the map for resolving
+inputs:
+  docfx.yml: |
+    xref: 
+      - xrefmap.json
+  docs/a.md: Link to @System.String
+  xrefmap.json: |
+    [{
+        "uid": "System.String",
+        "name": "String",
+        "fullName": "System.String",
+        "href": "https://ppe.docs.microsoft.com/e2eppe-core-docs/api/system.string",
```

Comment:
>https://ppe.docs.microsoft.com/e2eppe-core-docs/api/system.string [](start = 17, length = 65)

minor: this could be `[https://docs.microsoft.com/en-us/dotnet/api/system.string`](https://docs.microsoft.com/en-us/dotnet/api/system.string`), which is the public URL also available to public